tethys_gizmos/urls.py

```diff
@@ -23,8 +23,7 @@
 
 urlpatterns = [
     url(r'^$', gizmo_showcase_views.index, name='showcase'),
-    url(r'^editable-map/$', gizmo_showcase_views.editable_map, name='editable_map'),
-    url(r'^google-map/$', gizmo_showcase_views.google_map, name='google_map'),
+    url(r'^google-map-view/$', gizmo_showcase_views.google_map_view, name='google_map_view'),
```

Comment:
I was going to ask you about this, but you had to go pretty quick.  We should discuss this.